src/oss.yml

```diff
@@ -166,6 +166,10 @@ paths:
     servers:
       - url: "/private"
     $ref: "./oss/paths/legacy_authorizations_authID_password.yml"
+  /legacy/query:
```

Comment:
The query path needs to be `/query` with no `/legacy/` or `/api/v2/` in front.
Since this would be a duplicate path as the `/query` under `/api/v2`, we may need to add `/api/v2` to all the paths manually to avoid the path collision.

Are the other changes in files here autogenerated?